flake.nix

```diff
@@ -555,8 +551,8 @@
 
       devShells = perSystem (system:
         {
-          "ghc810" = self.ghc810Flake.${system}.devShell;
           "ghc9" = self.flake.${system}.devShell;
+          "ghc810" = self.ghc810Flake.${system}.devShell;
```

Comment:
This should be removed

Reply:
I just thought it doesn't really matter as there's nothing in there really that has to be built extra. (Hls comes from nixpkgs and is reliably cached) but if you wish I'll remove it. ;)

Reply:
A whole extra GHC potentially, and all the libraries we depend on. That's a lot.